wp-offline-shell/readme.txt

```diff
@@ -8,11 +8,11 @@ Stable tag: 0.3.0
 License: GPLv2 or later
 License URI: http://www.gnu.org/licenses/gpl-2.0.html
 
-A WordPress plugin for caching theme assets via a service worker for the sake of performance and offline functionality.
+A WordPress plugin for caching theme assets via the [ServiceWorker API](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API/Using_Service_Workers) to improve performance and provide offline functionality.
 
 == Description ==
 
-Offline Shell is a WordPress plugin for using the [ServiceWorker API](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API/Using_Service_Workers "Learn more") to cache theme files for performance and offline purposes.
+Offline Shell is a WordPress plugin which utilizes the client side [ServiceWorker API](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API/Using_Service_Workers "Learn more") to cache theme files for performance and offline purposes.
```

Comment:
To differentiate it from the server-side plugins which WordPress has in abundance.